### DISCUSSION TITLE
Feature suggestion - sorting phrases of terms of service

### DISCUSSION THREAD

**Discussion by AbeNMS92** [tosdr/CrispCMS#76] [Ideas] [answered]
When I click on the notification, I noticed that the terms of service phrases (though very helpful) are not organized (from what I can see). I'm rather new to ToSDR but could we have maybe a dropdown menu to sort by urgency (gray, red, yellow, green color), by order listed in website ToS, and by privacy type?
<img width="1440" alt="Screen Shot 2021-04-07 at 10 07 08" src="https://user-images.githubusercontent.com/36974424/113879980-0e3b1980-9789-11eb-9820-072c0dc19ee2.png">

**Comment by JustinBack** [comment, marked as answer]
I have added it to the roadmap
https://git.tosdr.org/tosdr/roadmap/issues/10

We used to display those ratings by weight, that somehow has been removed in a commit
